things/game.rb

```diff
@@ -0,0 +1,16 @@
+require 'date'
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer, last_played_at)
```

Comment:
Kindly initialize the parent class argument (publish_date) or by using *args

Reply:
Thanks, @michael-duke  for the honest review. I have fixed the issues as suggested and we are good to go. Kindly re-review it again for your approval. Thanks

🚀